metapool/tests/test_metapool.py

```diff
@@ -80,6 +81,7 @@ def setUp(self):
         self.blanks = pd.read_csv(blanks_fp, sep='\t')
         self.sa_df = pd.read_csv(sa_fp, sep='\t',
```

Comment:
I think wrapping these read_csv calls with these specific parameters might be better off inside a wrapper-function like I mentioned earlier. It's not clear to me whether it's okay that just TubeCode is being casted to str this time or not.

Reply:
that specific self.blanks object is not from the newly added tests. We import BLANK and Katharoseq controls from directory using the add_controls() function.